### DISCUSSION TITLE
would we publish a jsr package for rolldown on jsr.io?

### DISCUSSION THREAD

**Discussion by Cecil0o0** [rolldown/rolldown#1024] [Ideas]
[jsr](https://jsr.io/) means JavaScript Registry, which is superset of npm. JSR is a modern package registry for JavaScript and TypeScript. JSR works with many runtimes (Node.js, Deno, browsers, and more) and is backwards compatible with npm.

maybe make it into a cd flow is a good idea, would we add a sub-flow in rolldown's workflow?

**Comment by hyfdev** [comment]
Yes. We have registered the `rollldown` scope.
